web/angular-wallet/package.json

```diff
@@ -94,7 +93,9 @@
     "@types/dragula": "^2.1.34",
     "@types/node": "~12.0.2",
     "codelyzer": "~5.0.1",
+    "cross-env": "^5.2.0",
     "electron": "^5.0.1",
+    "husky": "^2.3.0",
```

Comment:
husky is needed?

Reply:
yes, needed....I need to reinstall from time to time...I think it's usually sufficient to install only in one project (here: burstjs) -- but as it is devDep does not harm